pkgs/development/libraries/google-benchmark/default.nix

```diff
@@ -0,0 +1,88 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  # nativeBuildInputs
+  cmake,
+  ninja,
+  pkg-config,
+  # buildInputs
+  libpfm,
+  # checkInputs
+  filecheck,
+  gtest,
+  # Configuration options
+  buildSharedLibs ? true,
+  enableLTO ? true,
+  enableLibPFM ? true,
+}: let
+  inherit
+    (lib)
+    lists
+    strings
+    ;
+  setBool = bool:
+    if bool
+    then "ON"
+    else "OFF";
+  setBuildSharedLibrary = bool:
+    if bool
+    then "shared"
+    else "static";
+in
+  stdenv.mkDerivation (finalAttrs: {
+    strictDeps = true;
+    pname = "benchmark";
```

Comment:
There's already a Google benchmark package in Nixpkgs, under the name `gbenchmark`. Can you merge your extra options into the existing one?

Reply:
Good catch! Resolved in latest.